README.md

```diff
@@ -1 +1,40 @@
-# lookout-sonarcheck-analyzer
\ No newline at end of file
+# [![Build Status](https://travis-ci.org/src-d/lookout-sonarcheck-analyzer.svg)](https://travis-ci.org/src-d/lookout-sonarcheck-analyzer) lookout analyzer: sonarcheck
+
+A [lookout](https://github.com/src-d/lookout/) analyzer implementation that uses bblfsh UAST and a [sonar-checks](https://github.com/bblfsh/sonar-checks).
+
+_Disclamer: this is not official product, but only serves the purpose of testing the lookout._
```

Comment:
proposal to reword this:
this is not an official product, it only serves the purpose of testing lookout.